internal/status/routeconditions.go

```diff
@@ -39,6 +40,7 @@ const ReasonDegraded RouteReasonType = "Degraded"
 const ReasonValid RouteReasonType = "Valid"
 const ReasonErrorsExist RouteReasonType = "ErrorsExist"
 const ReasonGatewayAllowMismatch RouteReasonType = "GatewayAllowMismatch"
+const ReasonNoWeightedForwardTos = "NoWeightedForwardTos"
```

Comment:
nit: Should we name this `ZeroWeightedForwardTos` to say that they are explicitly zero and just not unset?